common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -231,12 +234,69 @@ public static boolean tryLoadItemList() {
                 .useCacheAsBackup()
                 .build());
 
-        handler.dispatch(false);
-
         // Check for success
         return isItemListLoaded();
     }
 
+    public static CompletableFuture<Boolean> tryLoadMap() {
```

Comment:
Actually, I think we should try to make clear that we are handling more than one map, so the method should be like `tryLoadMaps`, the cache should be like:
```
final File mapsDirectory = new File(API_CACHE_ROOT, "maps");
```

etc.

And the more I think about it, the stronger I feel that we should also correctly loop over all maps in the json response, and not just hard-code one map.

Reply:
Should be resolved.